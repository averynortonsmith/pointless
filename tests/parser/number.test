
y = 123
z = 4.5