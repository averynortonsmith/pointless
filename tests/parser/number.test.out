(Program 
 null
 []
 (Def 
  (Name "y")
  (Number 123))
 (Def 
  (Name "z")
  (Number 4.5)))
